translations/base-zh-TW.yaml

```diff
@@ -258,13 +258,13 @@ dialogs:
     upgradesIntroduction:
         title: 解鎖建築升級
         desc: >-
-            你生產過的所有圖形可以被用來升級建築。<strong>不要銷毀你之前建造的工廠！</strong>
-            升級選單在屏幕右上角。
+            </strong>你生產過的所有圖形都會被用來升級建築。升級菜單在畫面右上角。
```

Comment:
This is wrong, you start it with `</strong>`. Have a look at the original translation :)